spec/system/jobseekers_can_withdraw_a_job_application_spec.rb

```diff
@@ -0,0 +1,27 @@
+require "rails_helper"
+
+RSpec.describe "Jobseekers can withdraw a job application" do
+  let(:jobseeker) { create(:jobseeker) }
+  let(:vacancy) { create(:vacancy, organisation_vacancies_attributes: [{ organisation: build(:school) }]) }
+  let!(:job_application) { create(:job_application, :status_submitted, jobseeker: jobseeker, vacancy: vacancy) }
+
+  before do
+    allow(JobseekerApplicationsFeature).to receive(:enabled?).and_return(true)
+    login_as(jobseeker, scope: :jobseeker)
+    visit jobseekers_job_application_path(job_application)
+    click_on I18n.t("buttons.withdraw_application")
+  end
+
+  it "submits the form, renders error, then withdraws the application" do
+    click_on I18n.t("buttons.withdraw_application")
+
+    expect(page).to have_content("There is a problem")
+
+    choose "jobseekers-job-application-withdraw-form-withdraw-reason-other-field"
+
+    expect { click_on I18n.t("buttons.withdraw_application") }
```

Comment:
Not entirely sure, but if we test record creation in request specs, we can probably test only the interface in system spec, things that are on the screen.

Reply:
Yeah I'm with you on that actually. 

I'd like to keep system specs as clean and simple as possible across the whole application 👍